templates/taxi/car_detail.html

```diff
@@ -2,11 +2,23 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Car model: {{ car.model }}
+    <a href="{% url 'taxi:car-update' pk=car.id %}">
+      <i class="bi bi-pencil"></i>
+    </a>
+    <a href="{% url 'taxi:car-delete' pk=car.id %}">
+      <i class="bi bi-trash"></i>
+    </a>
+  </h1>
+  <hr>
+  <h5>Drivers:</h5>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
```

Comment:
The iteration over `car.drivers.all` is missing parentheses. It should be `car.drivers.all()` to correctly call the method and retrieve the list of drivers.